scipy/stats/stats.py

```diff
@@ -2896,9 +2896,9 @@ def spearmanr(a, b=None, axis=0):
         `axis`. Both arrays need to have the same length in the `axis`
         dimension.
     axis : int or None, optional
-        If axis=0 (default), then each column represents a variable, with
-        observations in the rows. If axis=0, the relationship is transposed:
-        each row represents a variable, while the columns contain observations.
+        If axis=0 (default), then each row represents a variable, with
+        observations in the columns. If axis=1, the relationship is transposed:
+        each column represents a variable, while the rows contain observations.
         If axis=None, then both arrays will be raveled.
```

Comment:
I don't think this is correct. The default behavior does treat each column as a variable. Check the example given below this paragraph:

```py
>>> x2n=np.random.randn(100, 2)
>>> spearmanr(x2n)
(0.022106210621062106, 0.82718799420247646)
```

Two columns, two variables, one output. For three columns, we get a 3×3 output, as expected.

The only mistake is that the second option is also described as `axis=0`.